skyportal/utils/fits_display.py

```diff
@@ -0,0 +1,55 @@
+from astropy.io import fits
+import copy
+import io
+import matplotlib
+import matplotlib.pyplot as plt
+import pysedm
+import tempfile
+
+
+def get_fits_preview(
+    image_data,
+    figsize=(8, 8),
+    output_format='png',
+):
+    """
+    Return an image of fits data
+
+    Parameters
+    ----------
+    image_data : bytes
+        Bytes representation of the fits images
+    figsize : tuple, optional
+        Matplotlib figsize of the png created
+    output_format : str, optional
+        "pdf" or "png" -- determines the format of the returned plot
+    Returns
+    -------
+    data
+        Byte representation of the data for display
+
+    """
+
+    matplotlib.use("Agg")
+    fig = plt.figure(figsize=figsize, constrained_layout=False)
+
+    image_data_copy = copy.deepcopy(image_data)
```

Comment:
why the deep copy?

Reply:
It seems like the fits reading function actually changes the data.